app/controllers/councillor_contributions_controller.rb

```diff
@@ -69,6 +69,16 @@ def show
     end
   end
 
+  def mark_as_reviewed
```

Comment:
This action should actually be called 'toggle_reviewed' or something, because it actually does the opposite of the current label sometimes.